pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -95,6 +95,59 @@ type ClusterList struct {
 	Items           []Cluster `json:"items"`
 }
 
+// ConditionStatus indicates the status of a condition (true, false, or unknown).
+type ConditionStatus string
+
+// These are valid condition statuses. "ConditionTrue" means a resource is in the condition;
+// "ConditionFalse" means a resource is not in the condition; "ConditionUnknown" means kubernetes
+// can't decide if a resource is in the condition or not. In the future, we could add other
+// intermediate conditions, e.g. ConditionDegraded.
+const (
+	ConditionTrue    ConditionStatus = "True"
+	ConditionFalse   ConditionStatus = "False"
+	ConditionUnknown ConditionStatus = "Unknown"
+)
+
+// ConditionType is a valid value for APIServiceCondition.Type
+type ConditionType string
```

Comment:
Here https://github.com/Azure/ARO-RP/blob/master/vendor/k8s.io/api/core/v1/types.go#L2520

Reply:
I understand this is for the operator, but it is running in a pod, so it is a podcondition.